![header v2](https://user-images.githubusercontent.com/47393421/151243992-4e65ad82-7ccc-4841-b403-cd5ee6bb751c.svg)

[![Linux](https://svgshare.com/i/Zhy.svg)](https://svgshare.com/i/Zhy.svg)
[![macOS](https://svgshare.com/i/ZjP.svg)](https://svgshare.com/i/ZjP.svg)
[![Atom](https://badgen.net/badge/icon/atom?icon=atom&label)](https://atom.io)
[![GitHub](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com)
[![PyPi](https://badgen.net/badge/icon/pypi?icon=pypi&label)](https://https://pypi.org/)
[![Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/vinyard_m)
[![PyPI pyversions](https://img.shields.io/pypi/pyversions/annoyance.svg)](https://pypi.python.org/pypi/annoyance/)

I'm currently working on my PhD at Harvard University where I am developing methods for the analysis of single-cell genomics data. I also work on methods for CRISPR-Cas9-based genome-editing tools (like base-editing) to study cancer progression. My PhD work is funded by an NIH F31 award through the NCI and focuses on applying innovations in deep learning to analogous problems in the analysis of single-cell data. You can read more about me and my studies at my [website](https://www.michaelvinyard.com).

I am passionate about and interested in: 

- üßø  (single-cell) epigenomics and chromatin biology
- üß¨  Functional annotation of variation in the human genome
- üßÆ  Deep learning
- ‚úÇÔ∏è  Functional CRISPR studies
- üß†  Psychiatric disorders
- üß´  Cancer biology
- üêç  Implementing open-source tools for genomics in python.
- üé®  Computer graphics art

### Links

<a href="https://scholar.google.com/citations?hl=en&view_op=list_works&gmla=AJsN-F6rmMbElNkG3r5SvJcnJv6WkT-KChoMpaB9rer-Cgre2G6kPA3clgazm7pwAODm-0FUgmLTZZ75gH565IvCWnSkoHdZsf9aknoZyiXnZoBDXHaN3MQ&user=uj1_ksoAAAAJ"><img src="https://user-images.githubusercontent.com/47393421/142145409-04c70c23-71a9-4b8d-b2df-509e7ad658dc.png" alt="scholar-logo" width="25" hspace="5"/></a>   <a href="https://www.linkedin.com/in/michaelvinyard/"><img src="https://user-images.githubusercontent.com/47393421/142145774-4a8cefa7-f845-43c3-a36f-92ee747d69f8.png" alt="linkedin-logo" width="25" hspace="5"/></a><a href="mailto:mvinyard@broadinstitute.org"><img src="https://user-images.githubusercontent.com/47393421/142145916-0428098b-c524-4f28-973b-775624becd7f.png" alt="mail-logo" width="25" hspace="5"/></a><a href="https://twitter.com/vinyard_m"><img src="https://user-images.githubusercontent.com/47393421/142146109-0f0ef6b5-d1b9-4a13-b69b-fbedf9bc086c.png" alt="mail-logo" width="25" hspace="5"/></a><a href="https://orcid.org/0000-0001-8167-1807"><img src="https://user-images.githubusercontent.com/47393421/142146398-bcdbfc40-3646-45ec-ad13-4c41c955f983.png" alt="orcid-logo" width="25" hspace="5"/></a>


### Repos and software
---
<a href="https://github.com/mvinyard/vintools/"><img src="https://user-images.githubusercontent.com/47393421/142250811-109c90c3-617f-43d0-918c-698894eddbea.png" alt="vintools-logo" height="60" hspace="10"/></a><a href="https://github.com/mvinyard/vinplots/"><img src="https://user-images.githubusercontent.com/47393421/142250672-7bdd2356-55dc-4fd3-a78a-ed41c7a2835b.png" alt="vinplots-logo" height="70" hspace="10"/></a><a href="https://github.com/mvinyard/cell-tools/"><img src="https://user-images.githubusercontent.com/47393421/142247961-78d4a367-acf6-4d59-9c6d-1104f25e8b01.png" alt="cell-tools-logo" height="70" hspace="10"/></a><a href="https://github.com/mvinyard/seq-toolkit/"><img src="https://user-images.githubusercontent.com/47393421/151608439-549937d0-cd35-43f1-88b6-03a4fa839571.svg" alt="seq-toolkit-logo" height="85" hspace="20"/></a>

<a href="https://github.com/mvinyard/pyLaCroix/"><img src="https://user-images.githubusercontent.com/47393421/142250929-f38a5141-8a8a-4554-a0ad-6f198ed653ae.png" alt="pyLaCroix-logo" height="60" hspace="30"/></a><a href="https://github.com/mvinyard/annoyance/"><img src="https://user-images.githubusercontent.com/47393421/143382656-d283e09e-dd34-4386-bc71-0fcac164eaa0.png" alt="annoyance-logo" height="70" hspace="0"/></a><a href="https://github.com/mvinyard/licorice/"><img src="https://user-images.githubusercontent.com/47393421/143731976-1fc9816d-38fe-4ff0-a594-9a7374f0333d.png" alt="licorice-logo" height="50" hspace="20"/></a><a href="https://github.com/mvinyard/sglib/"><img src="https://user-images.githubusercontent.com/47393421/151607675-f9b897d1-4709-4313-b756-8527234bb159.svg" alt="sglib-logo" height="70" hspace="20"/></a>

<br></br>


### GitHub Stats
---
<a href="https://github.com/mvinyard">
  <img align="top" src="https://github-readme-stats.vercel.app/api?username=mvinyard&count_private=true&show_icons=true" />
</a>
<a href="https://github.com/mvinyard">
  <img align="top" src="https://github-readme-stats.vercel.app/api/top-langs/?username=mvinyard&hide=jupyter%20notebook&layout=compact&)](https://github.com/mvinyard/" />
</a>

### Selected Publications and preprints
---
#### [CRISPR-suppressor scanning reveals a nonenzymatic role of LSD1 in AML](https://www.nature.com/articles/s41589-019-0263-0)
* **Michael E. Vinyard**, et al. . *Nat. Chem. Biol.* DOI: 10.1038/s41589-019-0263-0 (**2019**).

#### [Assessement of computational methods for the analysis of single-cell ATAC-seq data](https://link.springer.com/article/10.1186/s13059-019-1854-5)
* Chen, H., Lareau, C.‚Ä†, Andreani, T.‚Ä†, **Michal E. Vinyard**‚Ä†, et al. . *Genome Biol.* DOI: 10.1186/s13059-019-1854-5 (**2020**).
* [GitHub](https://github.com/pinellolab/scATAC-benchmarking/)

#### [SIMBA: SIngle-cell eMBedding Along with features](https://www.biorxiv.org/content/10.1101/2021.10.17.464750v2)
* Huidong Chen, Jayoung Ryu, **Michael E. Vinyard**, Adam Lerer, & Luca Pinello. . _bioRxiv_, DOI: 2021.2010.2017.464750 (**2021**).
* [GitHub](https://github.com/pinellolab/simba)
* [docs](https://simba-bio.readthedocs.io/en/latest/)

#### [Current progress and potential opportunities to infer single-cell developmental trajectory and cell fate](https://www.sciencedirect.com/science/article/abs/pii/S2452310021000093)
* L. Wang‚Ä†, Q. Zhang‚Ä†, Q. Qin‚Ä†, N. Trasanidis‚Ä†, **Michael E. Vinyard**‚Ä†, et al., . *Curr. Opin. in Syst. Biol.* ISSN 2452-3100. DOI: 10.1016/j.coisb.2021.03.006 (**2021**).

#### [singlecellVR: interactive visualization of single-cell data in virtual reality](https://www.frontiersin.org/articles/10.3389/fgene.2021.764170/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Genetics&id=764170)
* Stein, D.‚Ä†, Chen, H.‚Ä†, Vinyard, M.E.‚Ä†, et al. . *Front. genet.* DOI: 10.1101/2020.07.30.229534 (**2021**).
* [GitHub](https://github.com/pinellolab/singlecellvr)
* [singlecellVR.com](https://singlecellvr.pinellolab.partners.org/)

---
[![made-with-Markdown](https://img.shields.io/badge/Made%20with-Markdown-1f425f.svg)](http://commonmark.org)
[![GitHub forks](https://img.shields.io/github/forks/mvinyard/mvinyard.svg?style=social&label=Fork&maxAge=2592000)](https://GitHub.com/mvinyard/mvinyard)
[![GitHub stars](https://img.shields.io/github/stars/mvinyard/mvinyard.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/mvinyard/mvinyard)

*Dyanmically generated stats are implemented from [GitHub Readme Stats](https://github.com/anuraghazra/github-readme-stats)*

_‚Ä† denotes equal contribution_
